cli/tsc/97_ts_host.js

```diff
@@ -768,9 +761,6 @@ for (const [key, value] of Object.entries(hostImpl)) {
   }
 }
 
-// @ts-ignore Undocumented function.
```

Comment:
We should try to lean off `@ts-ignore` and instead just remove the `/** @internal */`s in TypeScript's codebase. I updated https://github.com/denoland/TypeScript/pull/15 as part of this change.